test/controllers/MovementsControllerSpec.scala

```diff
@@ -73,6 +75,8 @@ class MovementsControllerSpec extends SpecBase with ScalaCheckPropertyChecks wit
   val localTime     = LocalTime.of(1, 1)
   val localDateTime = LocalDateTime.of(localDate, localTime)
 
+  val dateFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss'.'SS")
```

Comment:
Should we use this formatter explicitly in ResponseArrival? Also I think we can reference: `DateTimeFormatter.ISO_LOCAL_DATE_TIME`